src/shared/components/stepper/StepperBar.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components';
+import { barTypeStepCssMixin } from './stepperCss';
+import { StepStatusType, StepperContainerProps } from './types';
+import { determineStepStatus } from './utils';
+
+export const BarStepperContainer = ({ style, activeStep = 1, steps }: StepperContainerProps) => {
+  const activeStepIndex = Math.min(activeStep, steps.length) - 1;
+  const width = `${(activeStepIndex / (steps.length - 1)) * 100}%`;
```

Comment:
<img width="84" alt="스크린샷 2024-07-09 오후 8 42 56" src="https://github.com/TmaxFintech/corebank-demo-fe/assets/28257740/11cd3e0e-348b-4d42-a948-66b7e55232df">

```suggestion
  const isInitialStep = activeStepIndex === 0;
  const progressWidth = `${(activeStepIndex / (steps.length - 1)) * 100}%`;
  const width = isInitialStep ? '7px' : progressWidth;
```

Reply:
이게 더 자연스러워 보이네요! 한번 문의 해볼게요!